pkgs/by-name/no/nomacs/package.nix

```diff
@@ -32,7 +35,7 @@ stdenv.mkDerivation (finalAttrs: {
     cmake
     libsForQt5.wrapQtAppsHook
     pkg-config
-  ];
+  ] ++ lib.optional cudaSupport [ cudaPackages.cuda_nvcc ];
```

Comment:
s/`optional`/`optionals`/

(`Bool -> a -> List a` vs `Bool -> List a -> List a`)

Reply:
Thank you for your correction!